### DISCUSSION TITLE
How to set random_seed

### DISCUSSION THREAD

**Discussion by 11923303233** [open-mmlab/mmediting#1953] [How-to] [answered]
I want to compare the effect of different degradations on superresolution models. How to set the random seed to ensure that other contents are the same?

**Comment by zengyh1900** [comment, marked as answer]
You can set the random_seed in the config in default_runtime.py, e.g., https://github.com/open-mmlab/mmagic/blob/main/configs/_base_/gen_default_runtime.py#L3

You can check more details about randomness in https://mmengine.readthedocs.io/en/latest/common_usage/set_random_seed.html
